package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "http-zoo",
```

Comment:
We should probably come up with a more meaningful name...I'd call it something like `node-hamms` to keep credit and honour to the initial guy who made it. What do you think?